samples/ts-valueentity-shopping-cart/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "ts-valueentity-shopping-cart",
+  "version": "0.0.1",
+  "engines": {
+    "node": ">=13.0.0",
+    "npm": ">=6.0.0"
+  },
+  "main": "dist/index.js",
+  "dependencies": {
+    "@lightbend/akkaserverless-javascript-sdk": "0.33.0"
+  },
+  "devDependencies": {
+    "@lightbend/akkasls-scripts": "0.33.0",
+    "@types/mocha": "^9.0.0",
+    "chai": "^4.3.3",
+    "jsdoc": "^3.6.3",
+    "mocha": "^9.1.1",
+    "ts-mocha": "^8.0.0",
+    "uglify-js": "^3.7.7",
+    "typescript": "^4.3.5",
+    "rimraf": "^3.0.2"
+  },
+  "config": {
+    "dockerImage": "gcr.io/akkaserverless-public/samples-ts-value-entity-shopping-cart",
+    "sourceDir": "./src",
+    "testSourceDir": "./test",
+    "protoSourceDir": "./proto",
+    "generatedSourceDir": "./lib/generated",
+    "compileDescriptorArgs": []
+  },
+  "scripts": {
+    "start": "node dist/index.js",
+    "test": "ts-mocha test/**/*.test.ts",
+    "integration-test": "ts-mocha integration-test/**/*.integration-test.ts",
+    "build": "akkasls-scripts build && rimraf {src,test,integration-test}/**/*.js && rimraf lib/**/*{shoppingcart,index}.{js,d.ts} && tsc",
```

Comment:
@pvlugter the `rimraf` part is a workaround, because the `akkasls-scripts` generates `.js` files, which we have to cleanup on each build. 
Once the codegen becomes TS compatible we can remove this hack.

The cleanup on the `lib` is just small extra, because we don't need those files anymore.